docs/duration.md

```diff
@@ -54,7 +54,7 @@ For more detailed information, see the ISO 8601 standard or the [Wikipedia page]
 
 All of the arguments are optional.
 Any missing or `undefined` numerical arguments are taken to be zero, and all non-integer numerical arguments are rounded down to the nearest integer.
-Negative numbers are not allowed.
+Any non-zero arguments must all have the same sign.
 
 Use this constructor directly if you have the correct parameters already as numerical values, but otherwise `Temporal.Duration.from()`, which accepts more kinds of input and allows disambiguation behaviour, is probably more convenient.
```

Comment:
While we're here, this text is awkward. Too many commas. Suggested replacement: 

> Use this constructor directly if you have the correct parameters already as numeric values. Otherwise, `Temporal.Duration.from()` is usually more convenient because it accepts more kinds of input and allows disambiguation behaviour.